clang/lib/Driver/ToolChains/CommonArgs.cpp

```diff
@@ -740,7 +740,8 @@ bool tools::isTLSDESCEnabled(const ToolChain &TC,
     SupportedArgument = V == "desc" || V == "trad";
     EnableTLSDESC = V == "desc";
   } else if (Triple.isX86()) {
-    SupportedArgument = V == "gnu";
+    SupportedArgument = V == "gnu" || V == "gnu2";
+    EnableTLSDESC = V == "gnu2";
```

Comment:
If GCC is happy to accept `trad` and `desc` aliases, then we can accept the aliases; otherwise, I do not want to introduce another way to express the same thing.